torchaudio/functional/functional.py

```diff
@@ -971,3 +975,34 @@ def spectral_centroid(
                            device=specgram.device).reshape((-1, 1))
     freq_dim = -2
     return (freqs * specgram).sum(dim=freq_dim) / specgram.sum(dim=freq_dim)
+
+
+def apply_codec(waveform, sample_rate, format, channels_first=True, compression=None) -> Tensor:
```

Comment:
Can you reorder the arguments like `waveform, sample_rate, compression=None, format, channels_first=True`?

The expected usage will change the value for `compression`, while `channels_first` will not be changed (at all).